plugins/plugin-optimize/plugin.js

```diff
@@ -2,6 +2,8 @@ const fs = require('fs');
 const path = require('path');
 const glob = require('glob');
 const colors = require('kleur/colors');
+const {minify: minifyHtml} = require('html-minifier');
+const {minify: minifyCss} = require('csso');
```

Comment:
Neat! Haven’t used `csso`; mostly just `cssnano`. I think this simplifies things over adding PostCSS as a dependency for this package.